### DISCUSSION TITLE
error http 400 bad request

### DISCUSSION THREAD

**Discussion by nomada-rodrigo** [FossifyOrg/General-Discussion#222] [General]
Hi,
I get this error when there is an attempt to synch the calendars. I reviewed my phone settings as suggested by DAVx5.
Can you help me find out why it stopped syncing? 

Thanks 
Rod

**Comment by naveensingh** [comment]
This should have been fixed by https://github.com/SimpleMobileTools/Simple-Calendar/pull/2053

If you can reproduce this issue with the latest Fossify Calendar release [here](https://github.com/FossifyOrg/Calendar/releases/tag/1.0.3), please file a bug report at [this page](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml). Make sure to include the CalDAV server you are using and attach your events as ICS files if you can.

**Comment by nomada-rodrigo** [comment]
Hi,
Any news on this issue? 

Thanks
Rodrigo

**Reply by naveensingh** [reply]
I'll try to reproduce it later. Any updates will be posted on your bug report thread.
